### ISSUE TITLE
NTR: Inhibin A high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8375]
New term request Inhibin A high in blood (elthat_140106041550):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000519
- label: Elevated circulating inhibin A concentration
- parents: [Abnormal circulating inhibin level (HP:0031099)](https://hpo.jax.org/app/browse/term/HP:0031099)
- pmid: PMID:8940372
- def: The concentration of inibin A in the blood circulation is above the upper limit of normal.
- comment: Inhibin A and inhibin B are related dimeric protein hormones and endocrine regulators of the reproductive axis. Specifically, inhibin inhibits FSH secretion from the anterior pituitary. The inhibins are synthesized by the gonads and are themselves modulated by FSH.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8375
- synonyms:
